src/Runner/Runner.php

```diff
@@ -169,9 +169,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                 $fixer->fix($file, $tokens);
 
                 if ($tokens->isChanged()) {
+                    $appliedFixers[$fixer->getName()] = $tokens->findChangedLines();
```

Comment:
btw, this does change the output structure of `Runner::fix`, which is a BC breaker